### PR TITLE
Add elm-format 0.8.6

### PR DESCRIPTION
**Pull request by avh4**

### CODE REVIEW

File: scripts/TestAllDownloads.expected.txt [hunk 1]
```diff
@@ -96,12 +96,16 @@ TestAllDownloads/
       }
   5/
     elm-tooling/
+      elm-format/
+        0.8.6/
+          elm-format
```
Comment (avh4) [inline scripts/TestAllDownloads.expected.txt:101]: Is there a command to update this file automatically?

Comment (lydell) [reply scripts/TestAllDownloads.expected.txt:101]: Yes: https://elm-tooling.github.io/elm-tooling-cli/contributing/#adding-a-new-version

File: src/KnownTools.ts [hunk 2]
```diff
@@ -286,6 +272,20 @@ const knownTools = {
...
+        fileName: "elm-format",
+        type: "tgz",
+      },
+      "linux-x64": {
+        hash: "52ca8edb0274148215b22941aa7b0b9b23cfe73f47c0798cc8c848974d8ff6a3",
+        url: "https://github.com/avh4/elm-format/releases/download/0.8.6/elm-format-0.8.6-linux-x64.tgz",
+        fileSize: 4994220,
+        fileName: "elm-format",
+        type: "tgz",
+      },
```
Comment (avh4) [inline src/KnownTools.ts:288]: :+1: fixed my script for next time

File: scripts/TestAllDownloads.expected.txt (hunk 1 above)
Comment (avh4) [reply scripts/TestAllDownloads.expected.txt:101]: :+1: updated my script for next time

**Comment by lydell** [comment]
Released in v1.13.0.

**Pull request merged**